_posts/2021-08-26-geometric-deep-learning/geometric_deep_learning.Rmd

```diff
@@ -0,0 +1,125 @@
+
+In this situation, one may feel grateful for approaches that aim to elucidate, complement, or replace some of the magic. By "complement or replace", I'm alluding to attempts to incorporate domain-specific knowledge into the training process. Interesting examples exist in several sciences, and I certainly hope to be able to showcase a few of these, on this blog, at a later time. As to the "elucidate", this characterization is meant to lead on to the topic of this post: the program of *geometric deep learning*.
+
+## Geometric deep learning: An attempt at unification
+
+Geometric deep learning (henceforth: GDL) is what a group of researchers around Michael Bronstein, Joan Bruna, Taco Cohen, and Petar Velicković call their attempt to build a framework that places deep learning (DL) on a solid mathematical basis.
+
+Prima facie, this is a scientific endeavor: They take existing architectures and practices and show where these fit into the "DL blueprint". DL research being all but confined to the ivory tower, though, it's fair to assume that this is not all: From those mathematical foundations, it should be possible to derive new architectures, new techniques to fit a given task. Who, then, should be interested in this? Researchers, for sure; to them, the framework may well prove highly inspirational. Secondly, everyone interested in the mathematical constructions themselves --- this probably goes without saying. Finally, the rest of us, as well: Even understood at a purely conceptual level, the framework offers an exciting, inspiring view on DL architectures that -- I think -- is worth getting to know about as an end in itself. And to provide a high-level introduction is the goal of this post.
+
+Before we get started though, let me mention the primary source for this text: [Geometric Deep Learning: Grids, Groups, Graphs, Geodesics, and Gauges](https://arxiv.org/abs/2104.13478) (@GDL).
+
+### Geometric priors
+
+A *prior*, in the context of machine learning, is a constraint imposed on the learning task. A generic prior could come about in different ways; a *geometric* prior, as defined by the GDL group, arises, originally, from the underlying *domain* of the task. Take image classification, for example. The domain is a two-dimensional grid. Or graphs: The domain are collections of nodes and edges.
```

Comment:
You have "the domain is" in one sentence and "the domain are" in the latter. This sounds weird (since "the domain" is singular). For the second sentence you could do something like "The domain consists of collections of nodes and edges."